### PR TITLE
Add advanced settings to utilities

### PR DESCRIPTION
**Pull request by thisisdano**
Adding sections for 
- responsive variants
- state variants
- output control
- custom values

[Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-advanced-settings/utilities/color/)

### CODE REVIEW

**Comment by maya** [comment]
Some of the component pages are missing sidebars. For example: https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-advanced-settings/components/alerts/ doesn't have a sidebar.

**Comment by thisisdano** [comment]
Ooops. I didn't mean for that change to go into this PR, but now that it's here, I'll fix it!

**Comment by thisisdano** [comment]
OK, should be fixed once Federalist builds. Now I'm moving over to the branch I intended to do this work on in the first place...

File: pages/style/overview.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+layout: styleguide
+title: Style
+category: Style
+lead: We build System components from consistent global palettes of type scale, spacing units, and color. Your project’s custom theme palette provides a subset of the global palettes tailored to your project’s needs.
+type: docs
+---
+
+## The benefits of global style palettes
+
+<p>Our global palettes of <a href="{{ site.baseurl }}/style/type-scale/">type scale</a>, <a href="{{ site.baseurl }}/style/spacing-units/">spacing units</a>, and <a href="{{ site.baseurl }}/style/color/">color</a> are the visual building blocks any component. When we use these palettes consistently, not only do individual components fit better within the design of the larger website, but USWDS websites have better coherence across the entire federal government. This coherence provides a more familiar, reliable user experience.</p>
```
Comment (maya) [inline pages/style/overview.md]: Should this be in markdown like the following paragraphs?

Comment (thisisdano) [reply pages/style/overview.md]: Will the `<a>` links work in a markdown paragraph?

Comment (maya) [reply pages/style/overview.md]: You can use the link markdown format and it will work `[]()`

Comment (thisisdano) [reply pages/style/overview.md]: OK, cool. Dunno why I thought that wouldn't work...
One sec.

**Comment by maya** [comment]
The [grid column text](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-advanced-settings/utilities/layout-grid/) looks much bigger than it was before:
screen shot 2018-07-24 at 10 32 45 am

What I expect:
screen shot 2018-07-24 at 10 32 10 am

**Comment by maya** [comment]
[z-index](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-advanced-settings/utilities/display/#z-index)looks off at `z-500`
screen shot 2018-07-24 at 10 31 45 am

**Comment by thisisdano** [comment]
I think these new style errors were introduced with the `plugin --> palette` renaming update. I hadn't updated my npm package since the new release so I didn't see them. I'm updating my USWDS and updating the palette names and then these _should_ resolve.... 🤞

**Comment by thisisdano** [comment]
Ugh, found some bugs in the `2.0` release branch, so this will take a little bit more time...

**Comment by thisisdano** [comment]
Blocked by https://github.com/uswds/uswds/pull/2623
When this is merged, I'll update this branch.

**Pull request merged**